src/Microsoft.PowerShell.PSReadLine/ConsoleKeyChordConverter.cs

```diff
@@ -323,8 +323,16 @@ internal static char GetCharFromConsoleKey(ConsoleKey key, ConsoleModifiers modi
 
             // get corresponding character  - may be 0, 1 or 2 in length (diacritics)
             var chars = new char[2];
-            int charCount = NativeMethods.ToUnicode(
-                virtualKey, scanCode, state, chars, chars.Length, NativeMethods.MENU_IS_INACTIVE);
+            int charCount = 1;
+            try
+            {
+                charCount = NativeMethods.ToUnicode(
```

Comment:
I looked into putting the check in Readline.Initialize() but since it was a different class I would have to expose two additional members.  Seemed simpler to keep the state within the classes needing those apis.